vllm/config/compilation.py

```diff
@@ -629,15 +631,38 @@ def set_splitting_ops_for_v1(self):
 
     def set_splitting_ops_for_inductor_graph_partition(self):
         assert self.use_inductor_graph_partition
-        use_inductor_graph_partition_msg = (
-            "When use_inductor_graph_partition=True, splitting_ops "
-            "are ignored and set to an empty list. Instead, "
-            "\"tags=(torch._C.Tag.cudagraph_unsafe, ),\" is "
-            "used to annotate custom ops for graph partition.")
-        if self.splitting_ops is not None and \
-            len(self.splitting_ops) > 0:
-            logger.warning_once(use_inductor_graph_partition_msg)
-        self.splitting_ops = []
+
+        if self.splitting_ops:
+            logger.debug(
+                "Using splitting_ops=%s for inductor partition rules.",
+                self.splitting_ops,
+            )
+            return
+
+        if self.splitting_ops == []:
+            logger.warning_once(
+                "Empty splitting_ops provided with "
+                "use_inductor_graph_partition; defaulting to attention ops.")
```

Comment:
Empty should be allowed as well:
```suggestion
        if self.splitting_ops is not None:
            logger.debug(
                "Using splitting_ops=%s for inductor partition rules.",
                self.splitting_ops,
            )
            return
```